### DISCUSSION TITLE
Getting error websocket.js:39 WebSocket connection to 'ws://localhost:3000/socket.io/?EIO=4&transport=websocket' failed: again and again

### DISCUSSION THREAD

**Discussion by shashankgarg4** [socketio/socket.io#5025] [Q&A]
Hello i am using socket.io-client package in my frontend react project this is my frontend code 
![image](https://github.com/socketio/socket.io/assets/88320735/67dd6882-c6df-45c4-8034-9461b49ed07a)
and in backend my code is setup in node js this is my node js code so initially socket gets connected and prints in console "user is connected" then after sometime it shows error in console in loop
![image](https://github.com/socketio/socket.io/assets/88320735/b61b0e62-d8b2-4fc8-ab35-2141d535be9f)
![image](https://github.com/socketio/socket.io/assets/88320735/271aec5f-83a3-4547-b2ee-029d343dc181)
![image](https://github.com/socketio/socket.io/assets/88320735/e1c6a4e5-790b-4130-8e0a-17a0384b4580)
Can you please tell me why this error is coming thanks

**Comment by shashankgarg4** [comment]


**Reply by jokester** [reply]
@shashankgarg4 My crystal ball says you did something wrong.

Please learn to provide necessary information and ask nicely, maybe starting [here](http://www.catb.org/~esr/faqs/smart-questions.html).

Sorry but I have a life and a full time job. This question is not interesting enough to purchase my free tech support.

**Reply by shashankgarg4** [reply]
@jokester thank you for your feedback

**Reply by Debidutta-code** [reply]
Any one knows how to solve this

**Reply by shashankgarg4** [reply]
Hello @Debidutta-code did you also faced this error?

**Reply by Debidutta-code** [reply]
Yes I deployed my server on Vercel, and client on Netlify and I was getting that same continuous error on my console.

**Reply by shashankgarg4** [reply]
@Debidutta-code yes same issue
